packages/block-library/src/paragraph/test/paragraph.native.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Paragraph from '../edit';
+
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+jest.mock( '@wordpress/blocks' );
+
+const getTestComponentWithContent = ( content ) => {
+	return shallow(
+		<Paragraph
+			attributes={ { content } }
+			setAttributes={ jest.fn() }
+			onReplace={ jest.fn() }
+			insertBlocksAfter={ jest.fn() }
+		/>
+	);
+};
+
+const getTestInstanceWithContent = ( content ) => {
+	return getTestComponentWithContent( content ).instance();
+};
+
+describe( 'Paragraph block', () => {
+	it( 'renders without crashing', () => {
+		const component = getTestComponentWithContent( '' );
+		expect( component.exists() ).toBe( true );
+	} );
+
+	it( 'splits empty block on Enter', () => {
```

Comment:
Minor nitpick, specially since I think this is just a file move, but since there's no actually `Enter` keypress involved in this test, I wouldn't mention it on the test name

Reply:
Nice catch! Removed the reference to "on Enter".